.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: c
+addons:
+  postgresql: "9.6"
+  apt:
+    packages:
+      - postgresql-9.6
+      - postgresql-contrib-9.6
+      - postgresql-server-dev-9.6
+before_script:
+  - psql -f .travis/alter_system.sql
+  - sudo service postgresql restart 9.6
+script: make && sudo make install && make installcheck
+after_failure: cat regression.diffs
```

Comment:
This file can be pretty big and very unreadable (there is a test that generates a gigantic value to insert). Not sure if it will be very useful.

Reply:
It ends up being truncated, but I think it's still pretty helpful. (In my opinion, the diff formatting is the worst part — I haven't looked to see if there is a way to pass other `diff` flags here.)

Reply:
ok

Reply:
Oops, meant to link a specific example and forgot: https://travis-ci.org/getsentry/wal2json/builds/535429074